src/browser/request.ts

```diff
@@ -106,6 +106,7 @@ async function startFetchRequest(
     () => controller?.abort(),
   );
   const headers: { [key: string]: any } = {};
+  // @ts-ignore
```

Comment:
Use `@ts-expect-error` instead. Allows you to add a message and will throw an error if at some point this line _doesn't_ complain about types